wagtail/admin/localization.py

```diff
@@ -117,3 +122,40 @@ def get_available_admin_time_zones():
     return getattr(
         settings, "WAGTAIL_USER_TIME_ZONES", sorted(zoneinfo.available_timezones())
     )
+
+
+def get_localized_response(user, view_func, request, *args, **kwargs):
```

Comment:
We can simplify the signature a bit by dropping `user` and get it from `request.user`